templates/react/package.json

```
@@ -3,14 +3,17 @@
   "private": true,
   "scripts": {
     "build": "pnpm recursive run build",
-    "dev": "run-pty % pnpm dev:node % pnpm dev:client % pnpm dev:contracts",
+    "dev": "pnpm run initialize && pnpm recursive run dev",
     "dev:client": "pnpm --filter 'client' run dev",
     "dev:contracts": "pnpm --filter 'contracts' dev",
-    "dev:node": "pnpm --filter 'contracts' devnode",
+    "dev:node": "mud devnode",
+    "dev:pty": "run-pty % pnpm dev:node % pnpm dev:client % pnpm dev:contracts",
```

actually we probably want to add that here too
```suggestion
    "dev:pty": "pnpm run initialize && run-pty % pnpm dev:node % pnpm dev:client % pnpm dev:contracts",
```